### PR TITLE
Adds TypeScript typings.

### PR DESCRIPTION
**Pull request by torkelrogstad**
This PR adds TypeScript typings.

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        /**
+         * How often should we check inactivity
+         *
+         * Default: 2000
+         */
+        checkInterval?: number;
+        /**
+         * Callback for when the user becomes inactive
+         */
+        onInactivity: (...args: any[]) => any;
```
Comment (jkomyno) [inline index.d.ts]: `onInactivity` should be `onInactivity: (timeWentInactive: number) => void;`

**Comment by habibaz** [comment]
Any update on this? Really need the type script @torkelrogstad

**Comment by jkomyno** [comment]
Hi, @habibaz. I'm currently in the process of moving the version in the [dev](https://github.com/jkomyno/react-native-user-inactivity/tree/dev) branch to Typescript, providing types out of the box. This will deprecate this PR anyhow, since the inner API has changed. ETA ranging from some hours to a couple of days. Is it suitable for you?

**Comment by jkomyno** [comment]
@habibaz Please check out `react-native-user-inactivity@0.1.2-next`.
Closing this as this PR is deprecated.

**Comment by habibaz** [comment]
@jkomyno Thanks 💃

**Pull request closed without merging**